manifests/init.pp

```diff
@@ -65,7 +65,7 @@
     default => '',
   }
 
-  unless $version =~ Pattern[/\A(python)?[0-9](\.[0-9])+/,
+  unless $version =~ Pattern[/\A(python)?[0-9](\.?[0-9])+/,
         /\Apypy\Z/, /\Asystem\Z/, /\Arh-python[0-9]{2}(?:-python)?\Z/] {
     fail("version needs to be pypy, system or a version string like '3.5' or 'python3.5)")
```

Comment:
Now that this now allows python35, perhaps update the fail message?

Reply:
Updated.